Rakefile

```diff
@@ -4,6 +4,9 @@
 require 'bundler'
 require 'bundler/gem_tasks'
 require 'rake/testtask'
+
+PROJECT_DIR = File.expand_path(".")
```

Comment:
We need to add this to `www/Rakefile` as well since that is called independently of the project's Rakefile, but the www Rakefile calls the docs task which needs this constant as well.